programs/survival/item_holding_mobs_despawn_rule.sc

```diff
@@ -0,0 +1,21 @@
+// changes despawn rule for monster mobs. 
+// makes mobs who hold items despawnable as opposed to vanilla where they will never despawn.
+// mobs can only be made despawnable when they are given a custom name using nametags
+
+__command()->null;

```

Comment:
Did you want to add a way to manually trigger the command and forgot?

Reply:
I wanted to disable the command. but that failed. Same for the bottom case. I just want the user to load the app and the command to not appear in the command list.

Reply:
Is there a way I can disable it ?

Reply:
Just don't write the line for it, it won't be registered then.

Reply:
I have tried without the __command() line but it still shows up. :/